src/react-components/room/UserProfileSidebar.js

```diff
@@ -61,13 +89,11 @@ export function UserProfileSidebar({
               disabled={isNetworkMuted}
             />
             <Slider
```

Comment:
This sidebar component still seems fishy. The fact that its not getting any sort of value passed into it is a red flag (its storing state, which is being updated imperatively). Since its primary goal is essentially just restyling a native slider I would expect it to work almost exactly the same. I would expect something like this: 
```js
<Slider
      min={0}
      max={2}
      step={0.1}
      value={multiplier}
      onChange={updateMultiplier}
      className={styles.sliderInputContainer}
      disabled={isNetworkMuted || isMuted}
/>
```